### PR TITLE
Added my project Camera Adversaria to "Contestational tech projects" section

### PR DESCRIPTION
**Pull request by kieranbrowne**
Hey David,
A colleague pointed me to this repo. What a great resource!
I've added my project Camera Adversaria to the open source contestational tech section.
Hope that's okay
Kieran

### CODE REVIEW

**Comment by KOLANICH** [comment]
IMHO: if one really want to not being tracked, he just doesn't share photos of people faces, hands, fingers and records of voices. Adversarial examples can in principle be countered - people do it well, so machines can also do it. Likely the thing that is needed to be done is to generate enough adversarial examples and feed them into training set, and doing this obviously improves accuracy on unaltered images too. If your project found popularity, the result will be a lot of adversarial examples in the net.

So IMHO your project does more harm than benefit.

**Comment by kieranbrowne** [comment]
That may be a reasonable critique @KOLANICH, but it would surely apply to other projects included in this list. As anti-surveillance technologies become widely used, surveillance technologies tend to get better as a result, it's a sad fact.

Companies and researchers are trying to get around adversarial examples regardless. Why should we not use adversarial methods in defence against surveillance while they still work?

**Comment by KOLANICH** [comment]
>Why should we not use adversarial methods in defence against surveillance while they still work?

Becuse they almost guaranteedly will be broken in future. And maybe even in present - companies doing tracking have much larger datasts and resources to train on them than independent or academical researchers have. And maybe some unpublished results. What your app does is offloading costly augmentation of data for training the models to track people to the devises of the people who don't want to be tracked.

**Comment by kieranbrowne** [comment]
I'm not going to argue with you in this thread, you can read my justification for the project in [the paper](https://kieranbrowne.com/publications/camera_adversaria.pdf).
